atm.js

```diff
@@ -1,3 +1,148 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+
+  var savingsInput; //#savings .input
+  var checkingBalance=0; //#checking .balance
+  var savingsBalance=0;
+  var checkingInput;
+  var totalBalance;
+
+  ///FUNCTIONS TO DETERMINE ACCOUNT COLOR///
+
+  function getAccountColor() {
+    if (checkingBalance==0) {
+      $("#checking .balance").addClass("zero");
+    }
+
+    else {
+      $("#checking .balance").removeClass("zero");
+    }
+  }
+
+  function getAccountColorB() {
+    if (savingsBalance==0) {
+      $("#savings .balance").addClass("zero");}
+
+      else {
+        $("#savings .balance").removeClass("zero");
+      }
+    }
```

Comment:
You've got some weird curly-bracing going on here. Keep it consistent.

Also, if you find yourself naming things like `itemA`, `itemB`, `itemC` and so forth, you can almost definitely extract those into a function to stay DRY. For example:

```js
var balances = {
    checking: 0,
    savings: 0
}

function getAccountColor(acctName) {
    if (balances[acctName]==0) {
        $("#" + acctName + " .balance").addClass("zero");
    }else{
        $("#" + acctName + " .balance").removeClass("zero");
    }
}

getAccountColor("checking");
getAccountColor("savings");
```